app/check-in/page.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import CheckInModal from "../components/Check-In";
```

Comment:
theres no use client hook as you are importing usestate in this file, so it doesnt compile. fix

Reply:
Alright, I'm on it!